master_changes.md

```diff
@@ -25,6 +25,7 @@ users)
 
 ## Install
   * Fix `opam install <local_dir>` not updating pinned packages' metadata [#6209 @kit-ty-kate - fix #5567]
+  * Fix `opam install --deps-only/--show-action <local_dir>` not updating pinned packages' metadata [#6209 @kit-ty-kate - fix #5567]
```

Comment:
Also, the PR name should change to something more general, like *fix opam install sometimes not updating pinned packages' metadata*, it is no more only about `local dir as a target.

Reply:
why is it not only about local dir?

Reply:
My mistake, I meant that it is not only the command `opam install <local-dir>` but also combined with option `--dep-only` and `--show-action` that is fixed here. I updated the proposal.

Reply:
what about "Fix opam install on a local directory not updating pinned packages' metadata" ?

Reply:
PR title updated